packages/server/graphql/public/fields/invoices.ts

```diff
@@ -30,14 +30,30 @@ export const invoices: NonNullable<UserResolvers['invoices']> = async (
     return {edges: [], pageInfo: {hasNextPage: false, hasPreviousPage: false}}
   const manager = getStripeManager()
 
-  const [session, upcomingInvoice, invoices] = await Promise.all([
+  const [sessionRes, upcomingInvoiceRes, invoicesRes] = await Promise.allSettled([
     manager.stripe.billingPortal.sessions.create({
       customer: stripeId,
       return_url: makeAppURL(appOrigin, `me/organizations/${orgId}/billing`)
     }),
     manager.retrieveUpcomingInvoice(stripeId),
     manager.listInvoices(stripeId)
   ])
+  if (
```

Comment:
Is this invoices logic related to the org user logic?

Reply:
100% invoices, i wanted to test it on the parabol org & it kept breaking in dev mode with prod data, so this prevents it from throwing a fatal error